packages/core/src/interfaces/charts.ts

```diff
@@ -320,13 +320,12 @@ export interface RadarChartOptions extends BaseChartOptions {
  * options specific to combo bar charts
  */
 export interface ComboChartOptions extends AxisChartOptions {
-	comboChartTypes?: {
-		[ChartTypes.LINE]?: [],
-		[ChartTypes.STACKED_BAR]?: [],
-		[ChartTypes.GROUPED_BAR]?: [],
-		[ChartTypes.SIMPLE_BAR]?: [],
-		[ChartTypes.SCATTER]?: []
-	};
+	experimental?: boolean;
```

Comment:
```suggestion
```

we don't need the experimental flag - we are just using this so the storybook demos have the experimental notification.

Reply:
@natashadecoste Sure thing! This means I also do not need to define it in my type - then, as this is an 'storybook only property'?